lib/diagnostic.js

```diff
@@ -39,20 +51,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'cuttie',
+  species: 'collie',
+  breed: 'arabian white'
+  vocalization: 'meowwww',
+  vocalize: function() {
+    console.log('purrrrrrr')
```

Comment:
Ideally you want this method to reference the `vocalize` property that you defined above. Something like this: `return this.vocalization`